safari.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Safari
+Version = 5.1.7
+License = EULA
+Description = Apple Safari for Windows version 5.1.7
+Category = 5
+URLSite = https://archive.org/details/safari-5.1.7-windows
+URLDownload = https://archive.org/download/safari-5.1.7-windows/5.1.7_SafariSetup.exe
+SizeBytes = 38494576
+Icon = safari.ico
```

Comment:
You can omit this field, as the icon filename matches the txt filename.

https://reactos.org/wiki/RAPPS#Icon

```suggestion
```